ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -821,12 +822,16 @@ const PanelMain = fp.flow(
                     <LinkButton
                       style={{
                         ...styles.deleteLink,
-                        ...(disableControls || !runtimeExists
+                        ...(disableControls ||
+                        !canDeleteRuntime(currentRuntime?.status)
                           ? { color: colorWithWhiteness(colors.dark, 0.4) }
                           : {}),
                       }}
                       aria-label='Delete Environment'
-                      disabled={disableControls || !runtimeExists}
```

Comment:
there are many different ways to refer to runtimes and their statuses. "exists" includes an isVisible() check which excludes the ERROR state. This is appropriate in many cases, but not for deletion